client/src/components/PastaShow.js

```diff
@@ -28,16 +35,67 @@ const PastaShow = (props) => {
   useEffect(() => {
         getPasta()
     }, [])
-  
+
+    const postReview = async (newReviewData) => {
+        try {
+            const response = await fetch (`/api/v1/pastas/${pastaId}/review`, {
+                method: 'POST',
+                headers: new Headers({
+                    'Content-Type': 'application/json'
+                }),
+                body: JSON.stringify(newReviewData)
+            })
+            if (!response.ok) {
+                if(response.status === 422) {
+                    const body = await response.json()
+                    const newErrors = translateServerErrors(body.errors)
+                    return setErrors(newErrors)
+                } else {
+                    const errorMessage = `${response.status} (${response.statusText})`
+                    const error = new Error(errorMessage)
+                    throw(error)
+                }
+            } else {
+                const body = await response.json()
+                const updatedReviews = pasta.reviews.concat(body.reviews)
+                setErrors([])
+                setPasta({...pasta, reviews: updatedReviews})
+                }
+        } catch(error) {
+            console.error(`Error in fetch: ${error.message}`)
+        }
+    }
+
+    let reviewTiles
+    if (!pasta.reviews[0]) {
+        reviewTiles = `There are currently no reviews for this pasta!`
```

Comment:
backticks are needed when we are string interpolating, we can use regular quotes here